Documentation/ceph-fs-mirror-crd.md

```diff
@@ -0,0 +1,46 @@
+---
+title: FilesystemMirror CRD
+weight: 3500
+indent: true
+---
+
+# Ceph FilesystemMirror CRD
+
+Rook allows creation and updating fs-mirror daemon through the custom resource definitions (CRDs).
+CephFS images can be asynchronously mirrored between two Ceph clusters.
+CephFS will support asynchronous replication of snapshots to a remote CephFS file system via cephfs-mirror tool.
+Snapshots are synchronized by mirroring snapshot data followed by creating a snapshot with the same name (for a given directory on the remote file system) as the snapshot being synchronized.
+For more information about user management and capabilities see the [Ceph docs](https://docs.ceph.com/en/latest/dev/cephfs-mirroring/#cephfs-mirroring).
+
+## Creating daemon
+
+To get you started, here is a simple example of a CRD to deploy an cephfs-mirror daemon.
+
+```yaml
+apiVersion: ceph.rook.io/v1
+kind: CephFilesystemMirror
+metadata:
+  name: my-fs-mirror
+  namespace: rook-ceph
+```
+
+### Prerequisites
+
+This guide assumes you have created a Rook cluster as explained in the main [Quickstart guide](ceph-quickstart.md)
+
+## Settings
+
+If any setting is unspecified, a suitable default will be used automatically.
+
+### FilesystemMirror metadata
+
+* `name`: The name that will be used for the Ceph cephfs-mirror daemon.
+* `namespace`: The Kubernetes namespace that will be created for the Rook cluster. The services, pods, and other resources created by the operator will be added to this namespace.
+
+### FilesystemMirror Settings
```

Comment:
Do we not need the filesystem name that will be mirrored? Or the daemon will look for all filesystems to mirror if there are multiple?

Reply:
The daemon looks up for available filesystems.